src/analysis/retail/shaman/enhancement/modules/talents/ElementalBlast.tsx

```diff
@@ -67,10 +67,11 @@ class ElementalBlast extends BaseElementalBlast {
       {},
     );
 
-    this.maxCharges = this.selectedCombatant.getMultipleTalentRanks(
-      TALENTS.ELEMENTAL_BLAST_ENHANCEMENT_TALENT,
-      TALENTS.LAVA_BURST_TALENT,
-    );
+    this.maxCharges = 1;
+    // this.maxCharges = this.selectedCombatant.getMultipleTalentRanks(
+    //   TALENTS.ELEMENTAL_BLAST_ENHANCEMENT_TALENT,
+    //   TALENTS.LAVA_BURST_TALENT,
+    // );
```

Comment:
if this is now truly 1 charge, should just remove the commented section

Reply:
It is currently bugged in prepatch and has 1 charge, so this was only meant to be temporary. I'll revert this

Reply:
ooooh that makes sense